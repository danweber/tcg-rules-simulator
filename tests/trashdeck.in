VERSION:2
# [When Attacking] Trash the top 3 cards of your deck.
FIELD1:Attacker1,DW1-Van,DW1-Trasher Attacker2,DW1-Van,DW1-Trasher Attacker3,DW1-Van,DW1-Trasher Noticer,DW1-Birder
DECK1:CS1-02 CS1-03 CS1-04 CS1-05
TRASH1:CS1-06
SECURITY2:CS2-02 CS2-02 CS2-02 CS2-02
GAME:P1 T3 MAIN 3
DUMP:TRASH,DECK,DP

ECHO: -- attack, cards go to trash, noticer gives +100 DP to something
CMD:ATTACK Attacker1 Player
STEP:AUTOANSWER:CAPTURE
DUMP:TRASH,DECK,DP

ECHO: -- attack, cards go to trash, just 1 because we're empty, noticer gives +100 DP to something
CMD:ATTACK Attacker2 Player
STEP:AUTOANSWER:CAPTURE
DUMP:TRASH,DECK,DP

ECHO: -- attack, nothing goes to trash, noticer doesn't notice anything
CMD:ATTACK Attacker3 Player
STEP:AUTOANSWER:CAPTURE
DUMP:TRASH,DECK,DP

EXIT
