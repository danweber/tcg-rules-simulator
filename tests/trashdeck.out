FIELD1:Attacker1,DW1-Vanillamon,DW1-Trasher,5000DP Attacker2,DW1-Vanillamon,DW1-Trasher,5000DP Attacker3,DW1-Vanillamon,DW1-Trasher,5000DP Noticer,DW1-Birder,3000DP
FIELD2:
TRASH1:CS1-06
TRASH2:
DECK1:CS1-02 CS1-03 CS1-04 CS1-05
DECK2:
GAME:P1 T3 MAIN 3
ECHO: -- attack, cards go to trash, noticer gives +100 DP to something
CAPTURE:Target VanillaMon 1
CAPTURE:Target VanillaMon 2
CAPTURE:Target VanillaMon 3
CAPTURE:Target Birder 4
FIELD1:Attacker1,DW1-Vanillamon,DW1-Trasher,REST,5100DP Attacker2,DW1-Vanillamon,DW1-Trasher,5000DP Attacker3,DW1-Vanillamon,DW1-Trasher,5000DP Noticer,DW1-Birder,3000DP
FIELD2:
TRASH1:CS1-06 CS1-05 CS1-04 CS1-03
TRASH2:CS2-02
DECK1:CS1-02
DECK2:
GAME:P1 T3 MAIN 3
ECHO: -- attack, cards go to trash, just 1 because we're empty, noticer gives +100 DP to something
CAPTURE:Target VanillaMon 1
CAPTURE:Target VanillaMon 2
CAPTURE:Target VanillaMon 3
CAPTURE:Target Birder 4
FIELD1:Attacker1,DW1-Vanillamon,DW1-Trasher,REST,5200DP Attacker2,DW1-Vanillamon,DW1-Trasher,REST,5000DP Attacker3,DW1-Vanillamon,DW1-Trasher,5000DP Noticer,DW1-Birder,3000DP
FIELD2:
TRASH1:CS1-06 CS1-05 CS1-04 CS1-03 CS1-02
TRASH2:CS2-02 CS2-02
DECK1:
DECK2:
GAME:P1 T3 MAIN 3
ECHO: -- attack, nothing goes to trash, noticer doesn't notice anything
FIELD1:Attacker1,DW1-Vanillamon,DW1-Trasher,REST,5200DP Attacker2,DW1-Vanillamon,DW1-Trasher,REST,5000DP Attacker3,DW1-Vanillamon,DW1-Trasher,REST,5000DP Noticer,DW1-Birder,3000DP
FIELD2:
TRASH1:CS1-06 CS1-05 CS1-04 CS1-03 CS1-02
TRASH2:CS2-02 CS2-02 CS2-02
DECK1:
DECK2:
GAME:P1 T3 MAIN 3
